www/gatsby-node.js

```diff
@@ -465,7 +369,7 @@ exports.createPages = ({ graphql, actions, reporter }) => {
             context: {
               slug: node.fields.slug,
               locale,
-              ...nextAndPrev,
+              ...getPrevAndNext(node.fields.slug),
```

Comment:
Now that I think about it, what is the advantage of passing in this data in context instead of passing it the slug to the `<PrevAndNext>` component and calling `getPrevAndNext` from there? I don't think it relies on anything needed in gatsby-node.

Reply:
That seems like something for a followup PR, probably? Keep this scoped to moving the util?

Reply:
Yeah, that sounds good.